core/io/file_access_network.cpp

```diff
@@ -500,8 +500,11 @@ uint64_t FileAccessNetwork::_get_modified_time(const String &p_file) {
 void FileAccessNetwork::configure() {
 
 	GLOBAL_DEF("network/remote_fs/page_size", 65536);
+	ProjectSettings::get_singleton()->set_custom_property_info("network/remote_fs/page_size", PropertyInfo(Variant::INT, "network/remote_fs/page_size", PROPERTY_HINT_RANGE, "1, 65536, 1, or_greater")); //is used as denominator and can't be zero
 	GLOBAL_DEF("network/remote_fs/page_read_ahead", 4);
+	ProjectSettings::get_singleton()->set_custom_property_info("network/remote_fs/page_read_ahead", PropertyInfo(Variant::INT, "network/remote_fs/page_read_ahead", PROPERTY_HINT_RANGE, "0, 8, 1, or_greater"));
```

Comment:
We don't put a space after commas for hint range strings currently. It's not a strong convention but it's the predominant usage, so it's best to stick to it until we decide on a different convention.